compiler/stable_mir/src/mir/body.rs

```diff
@@ -64,6 +64,8 @@ type LocalDecls = Vec<LocalDecl>;
 pub struct LocalDecl {
     pub ty: Ty,
     pub span: Span,
+    pub local: Local,
```

Comment:
I don't think we need this, right?

Reply:
We definitely need this for getting `_0` values for locals

Reply:
The way the compiler does that, is by using enumerated iterator. We could add a few utility functions that return the pair local + local declaration